packages/opentelemetry-sdk-trace-base/test/common/export/SimpleSpanProcessor.test.ts

```diff
@@ -149,6 +149,14 @@ describe('SimpleSpanProcessor', () => {
   });
 
   describe('force flush', () => {
+    it('should call forceflush on exporter', () => {
+      const spyflush = sinon.spy(exporter, 'forceFlush');
+      const processor = new SimpleSpanProcessor(exporter);
+      processor.forceFlush().then(() => {
```

Comment:
I believe it would be better to await the returned promise here to be compliant with the API contract.